### ISSUE TITLE
Feature Request : Support needed for <frameset>

### ISSUE THREAD

**Issue by dipeshDM1** [DevExpress/testcafe#4085]
### What is your Test Scenario?
Swtiching to the frame to perfom operations on the elemetns preset inside it.

### What are you suggesting?
In our application, we have handled frame objects bu using <frame> tags (not <iframe>).
As we came to know, that Test cafe does not support <frame> elements (since, it is is marked obsolete in the W3C Recommendation), We are requesting you to handle <frame> element as well.
### What alternatives have you considered?
We tried other alternatives, but it didn't work

### Additional context
Node Version : v10.16.0
Testcafe version : 1.2.1

Note : We are looking forward to purchase this testcafe for automating our end-to-end automation flows. However, we might switch to another tool if this feature won't be available to us. We have raised the ticket "T802250" to devexpress support and they have suggested to tadd a new feature requst.

**Comment by devmondo** [comment]
Maybe i am wrong, but you could use `ClientFunction` to get the element using DOM selector and then pass it to the `TestController`? 

from the docs:

You can provide a function that takes arguments, and then pass serializable objects to the selector when you call it.
```js
import { Selector } from 'testcafe';

const elementWithId = Selector(id => {
    return document.getElementById(id);
});

fixture `My fixture`
    .page `http://www.example.com/`;

test('My Test', async t => {
    await t.click(elementWithId('buy'));
});
```

**Comment by alexey-lin** [comment]
@dipeshDM1, thank you for your suggestion.
 
Indeed, the frameset tag is obsolete and is not recommended for use. We would like to avoid adding support for elements that should [not to be used by web developers](https://www.w3.org/TR/html5-diff/#obsolete-elements)).
 
Support for outdated and obsolete technologies and functionality makes the product complex and hard to maintain. Refer to the following topic, which provides more information on these outdated elements: [Why were frames deprecated in HTML5, but not iFrames](https://softwareengineering.stackexchange.com/questions/144515/why-were-frames-deprecated-in-html5-but-not-iframes)?
 
It is also unknown what browsers support such outdated features as this functionality is not even tracked: https://caniuse.com/#search=frameset
 
We discussed your request with the team and decided not to add this feature. We looked for other tools that may support this obsolete functionality, but have not found any. As recommended by W3C, you might want to update your web application and avoid using outdated elements because they damage usability and accessibility.
